src/sentry/api/serializers/base.py

```diff
@@ -1,27 +1,27 @@
-from typing import Any, Mapping, Optional, Sequence, Union
+from typing import Any, Callable, List, Mapping, MutableMapping, Optional, Union
 
 import sentry_sdk
 from django.contrib.auth.models import AnonymousUser
 
 registry = {}
 
 
-def register(type: Any):
+def register(type: Any) -> Callable[[Any], Any]:
```

Comment:
Would use a typevar instead of any here: `Callable[[K], K]` and `K = TypeVar(...)`

typevar can probably be unbound, doesn't matter much